src/components/auth/profile/PersonalInfoTab.tsx

```diff
@@ -109,7 +120,7 @@ function PersonalInfoTab(props: { value: number; index: number }) {
               <p className={classes.bold}>парола:</p>
               <p>***********</p>
               <Box sx={{ position: 'absolute', right: '5px', top: '5px' }}>
-                <Link href="#">
+                <Link href="#" onClick={changePasswordHandler}>
```

Comment:
Since `next/link` component is designed to do in-app routing here we use it not the way it's supposed to

I'd suggest that we use another component `<ExternalLink href={link}>` which will open the password page in a new tab.